criu/pie/restorer.c

```diff
@@ -202,11 +202,31 @@ static int restore_creds(struct thread_creds_args *args, int procfd,
 	 * Setup supplementary group IDs early.
 	 */
 	if (args->groups) {
-		ret = sys_setgroups(ce->n_groups, args->groups);
-		if (ret) {
-			pr_err("Can't setup supplementary group IDs: %d\n", ret);
+		// We may be in an unprivileged user namespace where setgroups
```

Comment:
change to "... where we're not allowed to call setgroups." and remove the double space after the full stop.